ibm/service/power/data_source_ibm_pi_instance.go

```diff
@@ -162,6 +162,26 @@ func DataSourceIBMPIInstance() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			PIInstanceIbmiCSS: {
```

Comment:
I think computed attributes should not start with `pi_`.

Reply:
I don't necessarily disagree, but a lot of the computed attributes in the resource instance already start with pi_.

Reply:
IMO it is easy to read while working with the resource to differentiate between arg and attr. I will leave it to you.